packages/components/pie-input/src/index.ts

```diff
@@ -35,6 +35,9 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
     @property({ type: String })
     public name?: InputProps['name'];
 
+    @property({ type: Boolean })
```

Comment:
It may be worth reflecting this property. I believe we have done for the same property in other components and I think especially because it can be changed indirectly via fieldsets etc it would be worth doing. Let me know what you think!